README.md

```
@@ -2,223 +2,142 @@
+```
+</details>
 
-These overrides should be used with caution, especially the rate limiter bypass, as they can lead to exceeding actual API quotas if used against production services.
+#### References
+- [Java Docs](https://docs.oracle.com/en/java/index.html)
+- [Spring Boot Docs](https://docs.spring.io/spring-boot/docs/current/reference/html/)
+- [PMD Maven Plugin](https://maven.apache.org/plugins/maven-pmd-plugin/)
+- [SpotBugs Maven Plugin](https://spotbugs.github.io/)
+- [Maven Dependency Plugin](https://maven.apache.org/plugins/maven-dependency-plugin/)
 
 ## Troubleshooting
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)